index.js

```diff
@@ -82,11 +100,47 @@ cm.Item({
   }
 });
 
+// Vine context menu on Vine.co
+// only works from direct embed URLs structured like this: 
+// https://vine.co/v/[vineID] or https://vine.co/v/[vineID]/embed
+cm.Item({
+  label: "Send to mini player",
+  context: [
+  	cm.PredicateContext(checkLocation),
+  	cm.SelectorContext('[src*="mediasource:https://vine.co/"]')
```

Comment:
seems like with PredicateContext, we can get rid of SelectorContext altogether, right? I'm not sure why the video isn't loading from that link though. I'm looking into it.